lte/gateway/python/magma/pipelined/tests/test_uplink_bridge.py

```diff
@@ -298,6 +298,8 @@ def testFlowSnapshotMatch(self):
                                      include_stats=False)
 
 
+@unittest.skip
+# this reset default GW
```

Comment:
Why are we skipping these?

Reply:
this resets default route in the VM and mess up connectivity.

Reply:
I need to update the test to mock the route updates.